src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search--form search--form__big"
+      data-qa="big"
+    >
+      <label class="search--form__label search--form__label_big">
+        <input
+          type="text"
+          data-qa="keypress"
+          class="search--form__input search--form__input_big"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search--form search--form__small"
+      data-qa="small"
+    >
+      <label class="search--form__label search--form__label_small">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'search--form__small' seems to represent a style rather than the content meaning.